src/switch_core_media.c

```diff
@@ -11456,6 +11456,10 @@ static void generate_m(switch_core_session_t *session, char *buf, size_t buflen,
 		}
 	}
 
+	if (switch_channel_var_true(session->channel, "use_rtcp_xr") || (!switch_channel_var_false(session->channel, "use_rtcp_xr") && smh->mparams->offer_rtcp_xr)) {
```

Comment:
Should this be inside the validation of rtcp_audio_interval_msec? If we disable rtcp then we shouldn't send or offer rtcp-xr.